crates/relayer-framework-test/src/relayer_mock/base/types/aliases.rs

```diff
@@ -0,0 +1,15 @@
+use std::collections::HashMap;
+
+use crate::relayer_mock::base::types::chain::MockChainStatus;
+use crate::relayer_mock::base::types::height::Height as MockHeight;
+use crate::relayer_mock::base::types::state::State;
+
+pub type PacketUID = (PortId, ChannelId, Sequence);
+pub type ConsensusState = State;
+pub type ChainState = HashMap<MockHeight, ConsensusState>;
```

Comment:
Seems like there is some confusion here. The `ChainState` and `ConsensusState` should be just `State`.